--- a/upnpsoap.c
+++ b/upnpsoap.c
@@ -347,7 +347,7 @@ GetExternalIPAddress(struct upnphttp * h, const char * action, const char * ns)
 				ext_if_name);
 			ext_ip_addr[0] = '\0';
 		} else if (addr_is_reserved(&addr)) {
-			syslog(LOG_NOTICE, "private/reserved address %s is not suitable for external IP", ext_ip_addr);
+			syslog(LOG_DEBUG, "private/reserved address %s is not suitable for external IP", ext_ip_addr);
 			ext_ip_addr[0] = '\0';
 		}
 	}
